doc/glossary.rst

```diff
@@ -1336,7 +1341,12 @@ Methods
         often the likelihood of the data under the model.
 
     ``score_samples``
-        TODO
+        A method on an array of data points, which evaluates its predictions on
+        the given dataset, and returns an array consisting of log evaluations
+        for each.
+
+        It returns low values for high-dimensional data since evaluations are
```

Comment:
this is not true for outlier detection estimators.

Reply:
@albertcthomas Thanks so much for taking the time! This makes so much sense! I modified the definition for score_samples in the latest commit. Please let me know what you think